ui/src/pages/accounts/ConfirmEmail/ConfirmEmail.js

```diff
@@ -0,0 +1,44 @@
+import React, { Component } from 'react';
+import { Grid, Segment, Label } from 'semantic-ui-react';
+import { Loader } from '@components';
+import { parse } from '../utils';
+
+const Confirmed = ({ isConfirmed }) => {
+  console.log(isConfirmed);
+
+  return isConfirmed ? (
+    <Label color="green">Your email has been confirmed.</Label>
+  ) : (
+    <Label color="red">Your email couldn't be confirmed.</Label>
+  );
+};
+
+export default class ConfirmEmail extends Component {
+  componentDidMount() {
+    const params = parse(window.location.search);
+    this.props.confirmUser(params.token);
+  }
+
+  render() {
+    const { isConfirmed, isConfirmedLoading } = this.props;
+    return (
+      <Grid
+        textAlign="center"
+        verticalAlign="middle"
+        columns={2}
+        style={{ height: '100vh', backgroundColor: '#f9f9f9' }}
```

Comment:
For inline styles asks @kprzerwa I think there is an effort to remove every style from jsx, although it is more straight forward and convenient to style them in place.